site/src/Controller/OrganizationController.php

```diff
@@ -215,6 +220,23 @@ public function joinAction($organizationName, OrganizationRepository $repository
         $member->setUser($user);
         $member->setPermission(OrganizationMember::PERMISSION_REQUEST);
         $em = $this->getDoctrine()->getManager();
+        // Find Organisation members
+        $orgMembers = $repository->findOneByName($organizationName)->getMemberUsers();
```

Comment:
This should be limited to org admins, normal users cannot accept/reject a request to join.